jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/main/java/org/jbpm/console/ng/cm/backend/server/RemoteCaseManagementServiceImpl.java

```diff
@@ -47,7 +47,7 @@
 
     @Override
     public List<CaseDefinitionSummary> getCaseDefinitions() {
-        final List<CaseDefinition> caseDefinitions = client.getCaseDefinitions(0, Integer.MAX_VALUE);
+        final List<CaseDefinition> caseDefinitions = client.getCaseDefinitions(0, -1);
```

Comment:
What does page of size `-1` mean? I'd prefer named constat, like `PAGE_SIZE_UNLIMITED` to make it clear.